doc/devdocs/readme.md

```diff
@@ -75,12 +75,21 @@ modify --installpath "%ProgramFiles(x86)%\Microsoft Visual Studio\2019\%targetFo
 --add Microsoft.VisualStudio.Component.VC.ATL.Spectre
 ```
 
-### Compiling Source Code
+### Compiling Source Code from Visual Studio
 
 - Open `powertoys.sln` in Visual Studio, in the `Solutions Configuration` drop-down menu select `Release` or `Debug`, from the `Build` menu choose `Build Solution`.
 - The PowerToys binaries will be in your repo under `x64\Release`.
 - If you want to copy the `PowerToys.exe` binary to a different location, you'll also need to copy the `modules` and the `svgs` folders.
 
+### Compiling Source Code from the command line
+
+Run these commands in the Developer Command Prompt for Visual Studio or the Developer PowerShell for Visual Studio:
+```shell
+git submodule update --init --recursive
+msbuild -t:restore
+msbuild PowerToys.sln
+```
```

Comment:
These instructions are incomplete since you also need to setup the platform SDK.
I would suggest we don't add the msbuild instructions since it will also need to be maintain it and given is not a vital information, we should avoid creating work for ourselves.

Reply:
This PR will be a won't fix then?

Reply:
In general it would be better to first have the issue approved, so you won't spend time for something that may not get accepted.